### ISSUE TITLE
[RPC][transform][complete_constructor] Missing new line between generated properties

### ISSUE THREAD

**Issue by tkotosz** [phpactor/phpactor#1008]
After executing `transform: complete constructor` command the result looks like this:

![image](https://user-images.githubusercontent.com/4903365/80941006-967aa700-8de1-11ea-928e-9eedc91dc07c.png)

It is missing a new line between the first and second property. (Note: I am using a custom template to generate 1 line docblocks instead of the default multi-line one, but it seems to be unrelated to this issue)

**Comment by dantleech** [comment]
Yes this is a historical problem - the issue lies [here probably](https://github.com/phpactor/code-builder/blob/master/lib/Adapter/TolerantParser/Updater/ClassLikeUpdater.php#L51).

I have long wanted to replace this part of Phpactor with something better, but for now it would probably be enough to hack in this whitespace fix ....

**Comment by elythyr** [comment]
By memory the problem only occurs when you don't have any property in the class.
If you add some later it will work without issues.
And again, not sure but I think it will work without issues with phpactor configures to use PHP7.4 (because we don't generate the phpdoc anymore since we can type the attributes)
